server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from .shared_case import SharedTestCase
+from events.models import Event
+
+class EventTest(SharedTestCase):
+    """ Test module for Event model """
+
+    def setUp(self):
+        # event1
+        Event.objects.create(
+            name='event1',
+            creator=self.user1,
+            description='this is the description for event 1',
+            location='place1',
+            start_datetime=self.start_datetime1,
```

Comment:
Since the values themselves aren't really important, you might as well just write them in for more clarity.

Reply:
I'm assuming you're talking about start_datetime and end_datetime. Since I use timezone.now() for the start_datetime, I need to keep that value stored to calculate the end_datetime. Would you suggest I make datetime and timedelta objects in those lines?

Reply:
Yeah I would. I see that you changed that in a later commit. Nicely done.